gazebo_grasp_plugin/src/GazeboGraspGripper.cpp

```diff
@@ -10,6 +10,14 @@
 
 using gazebo::GazeboGraspGripper;
 
+#if GAZEBO_MAJOR_VERSION >= 8
+namespace gz_math = ignition::math;
+#define gz_math_Pose3d ignition::math::Pose3d
```

Comment:
Could we maybe do typedefs here instead of #defines? Like

```suggestion
typedef  ignition::math::Pose3d GzPose3d;
```